libs/utils/src/http/endpoint.rs

```diff
@@ -137,10 +137,14 @@ pub fn auth_middleware<B: hyper::body::HttpBody + Send + Sync + 'static>(
     })
 }
 
-pub fn check_permission(req: &Request<Body>, tenant_id: Option<TenantId>) -> Result<(), ApiError> {
+pub fn check_permission_with<E: Display>(
```

Comment:
E can/should be std::error:Error

Reply:
We use `anyhow::Error` instead. It does not implement `std::error::Error` for some reason. Changed to an explicit type.